.github/linters/.tflint_shared_services.hcl

```diff
@@ -10,7 +10,7 @@ plugin "azurerm" {
 }
 
 rule "terraform_typed_variables" {
-  enabled = false
+  enabled = true
 }
```

Comment:
This file is meant to only check the tags. Previously we had to turn off the types since that would cause an error (default is true) but now it's redundant and can be removed.